Python/specialize.c

```diff
@@ -452,6 +452,12 @@ initial_counter_value(void) {
 #define SPEC_FAIL_COMPARE_OP_EXTENDED_ARG 24
 
 /* FOR_ITER */
+#define SPEC_FAIL_FOR_ITER_REVERSED 4
```

Comment:
The values below 8 are common, in the section marked  `/* Common */` above.
You can always raise `SPECIALIZATION_FAILURE_KINDS` if you need.